src/main/java/edu/cornell/kfs/concur/batch/report/ConcurBatchReportLineValidationErrorItem.java

```diff
@@ -224,4 +233,12 @@ public void addItemErrorResult(String itemErrorResult) {
         this.itemErrorResults.add(itemErrorResult);
     }
 
+    public boolean getReportableAsLineLevelValidationError() {
```

Comment:
Since this is a getter for a boolean property, you may want to use "is" instead of "get" as the prefix.